runners/spark/pom.xml

```diff
@@ -134,6 +152,28 @@
             <forkCount>1</forkCount>
             <reuseForks>false</reuseForks>
           </configuration>
+          <executions>
+            <execution>
+              <id>runnable-on-service-tests</id>
+                 <configuration>
+                   <groups>org.apache.beam.sdk.testing.RunnableOnService</groups>
+                <parallel>none</parallel>
+                <failIfNoTests>true</failIfNoTests>
+                <dependenciesToScan>
+                  <dependency>org.apache.beam:java-sdk-all</dependency>
+                </dependenciesToScan>
+                <systemPropertyVariables>
+                  <beamTestPipelineOptions>
+                    [
+                      "--runner=org.apache.beam.runners.spark.TestSparkPipelineRunner",
+                      "--streaming=false"
+                    ]
+                  </beamTestPipelineOptions>
+                  <dataflow.spark.test.reuseSparkContext>true</dataflow.spark.test.reuseSparkContext>
```

Comment:
Well, let me check that, but take into account that it starts a standalone Spark node with WebUI server and all... 
And currently al tests run in a single thread, need to check if we could speed things up by increasing the number of threads without breaking some tests.

On it!